### PR TITLE
media-gfx/tuxpaint, media-gfx/tuxpaints-stamps: version bump, EAPI 8, cleanup

### PR DESCRIPTION
**Pull request by rndxelement**
I added two ebuilds for the latest versions of tuxpaint and tuxpaint-stamps with support for EAPI 8.
Notes:
- In tuxpaint I removed `xdg_src_prepare` from `src_prepare` because it is not exported anymore in EAPI 8
- In tuxpaint the sed call in `src_prepare` wasn't working fully, so I fixed it and moved it to a patch

### CODE REVIEW

File: media-gfx/tuxpaint/tuxpaint-0.9.27.ebuild [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+"
+DEPEND="${RDEPEND}"
+BDEPEND="
+	dev-util/gperf
+	media-gfx/imagemagick[jpeg,png,svg]
+	sys-devel/gettext
+"
+
+src_prepare() {
+	sed -i \
```
Comment (thesamesam) [inline media-gfx/tuxpaint/tuxpaint-0.9.27.ebuild:39]: This feels like it's begging for a proper patch, tbh.

File: media-gfx/tuxpaint/tuxpaint-0.9.27.ebuild [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+	media-libs/sdl-image[png]
+	media-libs/sdl-mixer
+	media-libs/sdl-pango
+	media-libs/sdl-ttf
+	x11-libs/cairo
+"
+DEPEND="${RDEPEND}"
+BDEPEND="
+	dev-util/gperf
+	media-gfx/imagemagick[jpeg,png,svg]
```
Comment (thesamesam) [inline media-gfx/tuxpaint/tuxpaint-0.9.27.ebuild:34]: virtual/imagemagick-tools instead, maybe?

File: media-gfx/tuxpaint-stamps/tuxpaint-stamps-20211125.ebuild [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+KEYWORDS="~amd64 ~ppc ~x86"
+
+RDEPEND="media-gfx/tuxpaint"
+DEPEND="${RDEPEND}"
+
+S=${WORKDIR}/${MY_P}
+
+src_prepare() {
+	default
+	eapply "${FILESDIR}"/${P}-make-382.patch
```
Comment (thesamesam) [inline media-gfx/tuxpaint-stamps/tuxpaint-stamps-20211125.ebuild:23]:
```suggestion

```

File: media-gfx/tuxpaint-stamps/tuxpaint-stamps-20211125.ebuild [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~ppc ~x86"
+
+RDEPEND="media-gfx/tuxpaint"
+DEPEND="${RDEPEND}"
+
+S=${WORKDIR}/${MY_P}
+
```
Comment (thesamesam) [inline media-gfx/tuxpaint-stamps/tuxpaint-stamps-20211125.ebuild:20]:
```suggestion

PATCHES=(
	${FILESDIR}"/${P}-make-382.patch
)

```

File: media-gfx/tuxpaint-stamps/tuxpaint-stamps-20211125.ebuild [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+SRC_URI="mirror://sourceforge/tuxpaint/${MY_P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~ppc ~x86"
+
+RDEPEND="media-gfx/tuxpaint"
+DEPEND="${RDEPEND}"
+
+S=${WORKDIR}/${MY_P}
```
Comment (thesamesam) [inline media-gfx/tuxpaint-stamps/tuxpaint-stamps-20211125.ebuild:19]:
```suggestion
```

File: media-gfx/tuxpaint-stamps/tuxpaint-stamps-20211125.ebuild [hunk 6]
```diff
@@ -0,0 +1,31 @@
...
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+MY_P=${PN}-${PV:0:4}.${PV:4:2}.${PV:6:2}
+
+DESCRIPTION="Set of 'Rubber Stamp' images which can be used within Tux Paint"
+HOMEPAGE="http://www.tuxpaint.org/"
+SRC_URI="mirror://sourceforge/tuxpaint/${MY_P}.tar.gz"
+
```
Comment (thesamesam) [inline media-gfx/tuxpaint-stamps/tuxpaint-stamps-20211125.ebuild:11]:
```suggestion
S=${WORKDIR}/${MY_P}

```

**Comment by rndxelement** [comment]
Thanks for the review!
I implemented your suggestions. Using imagemagick-tools instead of imagemagick seems to work.

**Pull request closed without merging**